lua/markdown.lua

```diff
@@ -512,8 +512,13 @@ function M.follow_link()
             -- an anchor
             vim.fn.search("^#* "..link.url:sub(2))
         else
-            -- a file
-            vim.cmd("e " .. link.url)
+            if string.find(link.url, '~') then
```

Comment:
Needs to be "^[~/]" I think. Only match beginning of string, and include paths like /home/... which are absolute from root